File: generated-sources/scala-finch/mojang-sessions/src/main/scala/org/openapitools/models/PlayerProfile.scala
package org.openapitools.models

import io.circe._
import io.finch.circe._
import io.circe.generic.semiauto._
import io.circe.java8.time._
import org.openapitools._
import org.openapitools.models.PlayerProfileProperty
import scala.collection.immutable.Seq

/**
 * 
 * @param id The player UUID without hyphens
 * @param name 
 * @param legacy Will appear in the response if the user has not migrated their minecraft.net account to Mojang.
 * @param properties An array with all player properties, like skin and cape
 */
case class PlayerProfile(id: String,
                name: String,
                legacy: Option[Boolean],
                properties: Seq[PlayerProfileProperty]
                )

object PlayerProfile {
    /**
     * Creates the codec for converting PlayerProfile from and to JSON.
     */
    implicit val decoder: Decoder[PlayerProfile] = deriveDecoder
    implicit val encoder: ObjectEncoder[PlayerProfile] = deriveEncoder
}

File: generated-sources/php-laravel/mojang-sessions/lib/app/Models/PlayerProfile.php
<?php
/**
 * PlayerProfile
 */
namespace app\Models;

/**
 * PlayerProfile
 */
class PlayerProfile {

    /** @var string $id The player UUID without hyphens*/
    private $id;

    /** @var string $name */
    private $name;

    /** @var bool $legacy Will appear in the response if the user has not migrated their minecraft.net account to Mojang.*/
    private $legacy;

    /** @var \app\Models\PlayerProfileProperty[] $properties An array with all player properties, like skin and cape*/
    private $properties;

}
